keyboards/kbdfans/kbdpad/mk1/mk1.h

```diff
@@ -13,26 +13,39 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
 #pragma once
 
 #include "quantum.h"
 
-// This a shortcut to help you visually see your layout.
-// The first section contains all of the arguments
-// The second converts the arguments into a two-dimensional array
-
-#define LAYOUT(   \
-  k52,  k53,           \
-  k40, k41, k42, k43,  \
-  k30, k31, k32,       \
-  k20, k21, k22, k23,  \
-  k10, k11, k12,       \
-  k00, k02, k03        \
+#define LAYOUT_ortho_6x4( \
+    K00,   K01,   K02,   K03,   \
+    K10,   K11,   K12,   K13,   \
+    K20,   K21,   K22,   K23,   \
+    K30,   K31,   K32,   K33,   \
+    K40,   K41,   K42,   K43,   \
+    K50,   K51,   K52,   K53    \
 ){ \
-  { k00,   KC_NO, k02, k03   }, \
-  { k10,   k11,   k12, KC_NO }, \
-  { k20,   k21,   k22, k23   }, \
-  { k30,   k31,   k32, KC_NO }, \
-  { k40,   k41,   k42, k43   }, \
-  { KC_NO, KC_NO, k52, k53   }, \
+    { K50,   K51,   K52,   K53   }, \
```

Comment:
why was the row order reversed here?

Reply:
Good question, I can undo this if you'd like. It made more sense to me (and matched other keebs I've seen) for K00 to be the top-right corner and KNM to be the bottom right order. But it literally just occurred to me it would be cleaner to do this by just reversing the MATRIX_ROW_PINS and MATRIX_COL_PINS arrays. So, two questions:

1. Is there a standard for numbering keys in a layout macro? In other words, was I correct to make K00 the top-left key, or is there actually no standard there? (I just pattern matched on the examples I've seen.)

2. If we do want K00 in the top left, should I just reverse the pin arrays instead of doing it this way?

Reply:
For the electrical matrix it is best practice to have the top left most key K00 following the notation of K row 0, col 0.

Reply:
Sorry, I just realized my question is ambiguous. :( I meant "should K00 be the top-left key physically on the keyboard", not the top-left key in this macro. Right now at HEAD K00 is the bottom-left key physically on the keyboard. This differed from what I've seen in other layout macros (where it's the top-left key physically on the keyboard), hence my change. (But I think the way I have this macro defined now is confusing, and just reversing the pins array would be a cleaner way to get K00 to be the top-left physical key.)

That said, as mentioned below, I think I should revert the layout stuff in this PR since it's a breaking change as I have it now, and that wasn't my intent.